app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java

```diff
@@ -36,21 +40,23 @@
 import butterknife.OnClick;
 import butterknife.OnItemSelected;
 import butterknife.OnTouch;
+import dagger.android.support.DaggerFragment;
 import fr.free.nrw.commons.R;
 import fr.free.nrw.commons.Utils;
-import fr.free.nrw.commons.di.CommonsDaggerSupportFragment;
 import fr.free.nrw.commons.settings.Prefs;
+import fr.free.nrw.commons.ui.widget.HtmlTextView;
 import timber.log.Timber;
 
 import static android.view.MotionEvent.ACTION_DOWN;
 import static android.view.MotionEvent.ACTION_UP;
 
-public class SingleUploadFragment extends CommonsDaggerSupportFragment {
+public class SingleUploadFragment extends DaggerFragment {
 
```

Comment:
Why did you changed this?

Reply:
Please undo it. Usage of `DaggerFragment` would cause crashes.

Reply:
Sure I will remove it @maskaravivek

Reply:
@maskaravivek, I had deleted the previous Commons forked repository due to some bugs and forked again. Can I make another PR as through this I'm unable to change my files?

Reply:
Should I make another PR that is in that I just have changed "upload/SingleUploadFragment.java" file as I had deleted the previously forked repo, I'm not also able to change this file directly from this PR.
@maskaravivek, @neslihanturan Please help!

Reply:
Or I have one solution for it, May you edit this @maskaravivek , @neslihanturan?

Reply:
@mithlesh4257 you can fork it again, and make a fresh PR don't worry. I am closing this. Please follow this document before https://github.com/commons-app/apps-android-commons/wiki/Setting-up-dev-enviroment by changing "maskaravivek" username with yours. And please make sure all of the steps we discussed here are covered before preparing PR. Make sure you checked your commits and added meaningful commit messages before pushing.

Besides, if you want to leave this task for someone else to handle, please notify us.